packages/eas-cli/src/build/createContext.ts

```diff
@@ -0,0 +1,138 @@
+import { Platform } from '@expo/eas-build-job';
+import { AndroidBuildProfile, BuildProfile, IosBuildProfile } from '@expo/eas-json';
+import JsonFile from '@expo/json-file';
+import resolveFrom from 'resolve-from';
+import { v4 as uuidv4 } from 'uuid';
+
+import { TrackingContext } from '../analytics/common';
+import { Analytics, BuildEvent } from '../analytics/events';
+import { CredentialsContext } from '../credentials/context';
+import { getExpoConfig } from '../project/expoConfig';
+import { getProjectAccountName, getProjectIdAsync } from '../project/projectUtils';
+import { resolveWorkflowAsync } from '../project/workflow';
+import { findAccountByName } from '../user/Account';
+import { ensureLoggedInAsync } from '../user/actions';
+import { createAndroidContextAsync } from './android/build';
+import { BuildContext, CommonContext } from './context';
+import { createIosContextAsync } from './ios/build';
+
+export default async function createBuildContextAsync<T extends Platform>({
```

Comment:
Maybe without `default` so all modules that import the function import it in the same way? It helps with refactoring and so on.